README.md

```diff
@@ -52,6 +54,17 @@ If you are new to Graviton and want to understand how to identify target workloa
 |DRAM	|8x DDR4	|8x DDR5	|
 |DDR Encryption	|yes	|yes	|
 
+# x86 monitoring differences
```

Comment:
this shouldn't be on the main page, maybe on a separate page with a paragraph about how different tools work differently on different architectures

Reply:
you could also have the output on another page